docs/src/unscented_kalman_inversion.md

```diff
@@ -1,12 +1,12 @@
 # Unscented Kalman Inversion
 
-One of the ensemble Kalman processes implemented in `EnsembleKalmanProcesses.jl` is the unscented Kalman inversion ([Huang, Schneider, Stuart, 2022](https://doi.org/10.1016/j.jcp.2022.111262)). The unscented Kalman inversion (UKI) is a derivative-free approximate Bayesian inference method that seeks to find the maximum a posteriori parameters ``\theta \in \mathbb{R}^p`` in the inverse problem
+One of the ensemble Kalman processes implemented in `EnsembleKalmanProcesses.jl` is the unscented Kalman inversion ([Huang, Schneider, Stuart, 2022](https://doi.org/10.1016/j.jcp.2022.111262)). The unscented Kalman inversion (UKI) is a derivative-free method for approximate Bayesian inference. We seek to find the posterior parameter distribution ``\theta \in \mathbb{R}^p`` from the inverse problem
 ```math
  y = \mathcal{G}(\theta) + \eta
 ```
 where ``\mathcal{G}`` denotes the forward map, ``y \in \mathbb{R}^d`` is the vector of observations and ``\eta \sim \mathcal{N}(0, \Gamma_y)`` is additive Gaussian noise. Note that ``p`` is the size of the parameter vector ``\theta`` and ``d`` is taken to be the size of the observation vector ``y``. The UKI algorithm has the following properties
 
-* UKI has a fixed ensemble size and requires only ``2p + 1`` particles in general.
+* UKI has a fixed ensemble size and requires only ``2p + 1`` particles, and form a quadrature stencil (rather than the random positioning of the particles from methods such as EKI)
```

Comment:
We actually have now two quadrature options - one with 2p+1 points and another `simplex` quadrature that requires p+2 if I remember correctly. It would be good to add that here.

Reply:
I've added this, and explained that the description below is only for the `symmetric` option